docs/core/porting/libraries.md

```diff
@@ -0,0 +1,348 @@
+.NET Framework 4.6.2 --> NET462
+.NET Standard 1.0    --> NETSTANDARD1_0
+.NET Standard 1.1    --> NETSTANDARD1_1
+.NET Standard 1.2    --> NETSTANDARD1_2
+.NET Standard 1.3    --> NETSTANDARD1_3
+.NET Standard 1.4    --> NETSTANDARD1_4
+.NET Standard 1.5    --> NETSTANDARD1_5
+.NET Standard 1.6    --> NETSTANDARD1_6
+```
+
+As mentioned above, if you are targeting a PCL, then you will have to specify a build definition for the compiler to understand.  There is no default definition that the compiler can use.
+
+### Using `project.json` in Visual Studio
+
+You have two options for using `project.json` in Visual Studio:
+
+1. A new xproj project type.
+2. A retargeted PCL project which supports .NET Standard.
+
+There are different benefits and drawbacks for each.
+
+#### When to Pick an Xproj Project
+
+The new Xproj project system in Visual Studio utilizes the capabilities of the `project.json`-based project model to offer two major features over existing project types: seamless multitargeting by building multiple assemblies and the ability to directly generate a NuGet package on build.
+
+However, it comes at the cost of lacking certain features you may use, such as:
+
+- Support for F# or Visual Basic
+- Generating satellite assemblies with localized resource strings
+- Directly referencing a `.dll` file on the filesystem
+- The ability to reference a csproj-based project in the Reference Manager (depending on the `.dll` file directly is supported, though)
+
+If your project needs are relatively minimal and you can take advantage of the new features of xproj, you should pick it as your project system.  This can be done in Visual Studio as such:
+
+1. Ensure you are using Visual Studio 2015 or later.
+2. Select File | New Project.
+3. Select ".NET Core" under Visual C#.
+4. Select the "Class Library (.NET Core)" template. 
+
+#### When to Pick a PCL project
+
+You can target .NET Core with the traditional project system in Visual Studio, by creating a Portable Class Library (PCL) and selecting ".NET Core" in the project configuration dialog.  Then you'll need to retarget the project to be based on the .NET Standard:
+
+1. Right-click on the project file in Visual Studio and select Properties.
+2. Under Build, select "Convert to .NET Standard".
+
+If you have more advanced project system needs, this should be your choice.  Note that if you wish to multitarget by generating platform-specific assemblies like with the `xproj` project system, you'll need to create a "Bait and Switch" PCL, as described in [How to Make Portable Class Libraries Work for You](https://blogs.msdn.microsoft.com/dsplaisted/2012/08/27/how-to-make-portable-class-libraries-work-for-you/).
+
+## Retargeting your .NET Framework Code to .NET Framework 4.6.2
+
+If your code is not targeting .NET Framework 4.6.2, it's highly recommended that you retarget.  This ensures that you can use the latest API alternatives for cases where the .NET Standard can't support existing APIs.
```

Comment:
"highly recommended". This seems a bit strong. Maybe just "recommended".

Also, 4.6.2 hasn't shipped as RTM yet!